live-examples/js-examples/expressions/expressions-logical-and-assignment.js

```diff
@@ -0,0 +1,9 @@
+let a = 1;
+
+a &&= 1;
+console.log(a);
+// expected output: 1
+
+a &&= 0;
+console.log(a);
+// expected output: 0
```

Comment:
As I read the docs, this operator behaves differently depending on the value of the left-hand side. But this example uses just one lhs value and two different rhs values. As such, it doesn't show the difference between `&&=` and plain `=`.

What if you have two different lhs values and one rhs value, like:

```js
let a = 1;
let b = 0;

a &&= 2;
console.log(a);
// expected output: 2

b &&= 2;
console.log(b);
// expected output: 0
```

Reply:
Yeah, that makes sense. I have this in the static example on the page, too.